### PR TITLE
Translations to Russian and Ukrainian

### PR DESCRIPTION
**Pull request by Larry0ua**
Update to #284

### CODE REVIEW

File: plugins/devices/frontend-java/src/main/java/com/freedomotic/jfrontend/MainWindow.java [hunk 1]
```diff
@@ -1500,7 +1500,7 @@ private void sendLogByMail() {
     private void jMenuItem1ActionPerformed(java.awt.event.ActionEvent evt) {
         String[] buttons = {i18n.msg("form_compilation"), i18n.msg("send_log"), i18n.msg("delete")};
         int option = JOptionPane.showOptionDialog(null, i18n.msg("report_issue_dialog"), i18n.msg("report_issue"),
-                JOptionPane.WARNING_MESSAGE, 0, null, buttons, buttons[2]);
+                JOptionPane.OK_CANCEL_OPTION, JOptionPane.ERROR_MESSAGE, null, buttons, buttons[2]);
```
Comment (Larry0ua) [inline plugins/devices/frontend-java/src/main/java/com/freedomotic/jfrontend/MainWindow.java]: By the way, this was changed as 3rd parameter of showOptionDialog is about buttons option, and 4th is about the style. I've changed parameters, but values remain the same - 2 and 0.

**Comment by mcicolella** [comment]
Hi, I have some conflicts with this PR. Please split the changes about translations and code.
About the code I didn't get exactly the problem so open a specific issue about it.
Thanks.

**Comment by Larry0ua** [comment]
updated

**Comment by Larry0ua** [comment]
Okay, actually that's not an issue, but a misprint, no functionality was affected by that specific change. I just reverted it

**Pull request closed without merging**